### PR TITLE
Documentation changes

### PR DESCRIPTION
**Pull request by shreyamalviya**
# What does this PR do?

Fixes some stuff in the documentation.

What's left:
- [x] fix targets where Agent is copied (`reference/agent_propagation.md`)
- [x] fix security report page (`reports/security.md`), mostly removing stuff and reordering/renaming sections
- [x] consider making changes to the content under `usage/configuration/`, and check that the content wherever these pages are referenced still makes sense
- [ ] screenshots!!!
- [ ] example report in `reports/security.md`

## PR Checklist
* [ ] Have you added an explanation of what your changes do and why you'd like to include them?
* [ ] Is the TravisCI build passing?
* [ ] Was the CHANGELOG.md updated to reflect the changes?
* [ ] Was the documentation framework updated to reflect the changes?
* [ ] Have you checked that you haven't introduced any duplicate code?

## Testing Checklist

* [ ] Added relevant unit tests?
* [ ] Have you successfully tested your changes locally? Elaborate:
    > Tested by {Running the Monkey locally with relevant config/running Island/...}
* [ ] If applicable, add screenshots or log transcripts of the feature working

### CODE REVIEW

File: docs/content/reference/exploiters/_index.md [hunk 1]
```diff
@@ -12,5 +12,3 @@ tags = ["reference", "exploit"]
 The Infection Monkey uses various remote code execution (RCE) exploiters. To our best knowledge, most of these pose no risk to performance or services on victim machines. This documentation serves as a quick introduction to the exploiters currently implemented and the vulnerabilities they use:
 
 {{% children %}}
-
-You can check out the exploiters' implementation yourself [in the Infection Monkey's GitHub repository](https://github.com/guardicore/monkey/tree/develop/monkey/infection_monkey/exploit).
```
Comment (cakekoa) [inline docs/content/reference/exploiters/_index.md:16]: Do we want to remove this link? I don't have an opinion on this, I just wanted to point out that it was removed.

Comment (cakekoa) [reply docs/content/reference/exploiters/_index.md:16]: From the commit:
> Hadoop is not a part of this so this is already outdated.
The rest of the exploiters will also soon be moved out of
here when we convert them to plugins.

File: docs/content/reference/scanners/_index.md [hunk 2]
```diff
@@ -1,47 +0,0 @@
----
-title: "Scanners"
-date: 2020-07-14T08:43:12+03:00
-draft: false
-weight: 20
-pre: '<i class="fas fa-network-wired"></i> '
-tags: ["reference"]
----
```
Comment (shreyamalviya) [inline docs/content/reference/scanners/_index.md:8]: Why did we drop this? We still have scanners and fingerprinters.

File: docs/content/reference/exploiters/_index.md (hunk 1 above)
Comment (VakarisZ) [reply docs/content/reference/exploiters/_index.md:16]: We'll have to have a definitive list where a user could go to to see the source code of the plugins/exploiters. I'd rather keep it and extend it with a link to Hadoop

File: docs/content/FAQ/_index.md [hunk 3]
```diff
@@ -157,27 +157,15 @@ If internet access is available, the Infection Monkey will use the internet for
...
-   suggest an update if one is available. No information gets collected during
-   this process.
-1. After the Monkey Island starts it sends a GET request to the analytics
-   server with your deployment type and a version number. This information gets
-   collected on the analytics server. It is used to understand which deployment
-   types/versions are no longer used and can be deprecated.
+1. While the Monkey Island Server is being set up, a GET request with the deployment
+type and version number is sent to the analytics server. This information is
+collected to understand which deployment types and versions are no longer used
+and can be deprecated.
```
Comment (VakarisZ) [inline docs/content/FAQ/_index.md:163]: I don't think this is true anymore

Comment (shreyamalviya) [reply docs/content/FAQ/_index.md:163]: It is. See Mike's comment above.

File: docs/content/usage/configuration/credentials.md [hunk 4]
```diff
@@ -0,0 +1,10 @@
+---
+title: "Credentials"
+date: 2020-06-09T12:20:08+03:00
+draft: false
+description: "Configure credentials that the Infection Monkey will use for propagation."
+---
+
+On this screen you can feed the Infection Monkey "stolen" credentials from your network, simulating an attacker with inside knowledge.
```
Comment (VakarisZ) [inline docs/content/usage/configuration/credentials.md:8]: I think these pages can be removed. They explain less then configuration descriptions, so I don't see how they would help the user

**Review by VakarisZ — APPROVED** [review]
See small suggestions. I didn't read through other changes, it's better we don't waste time arguig phrasing when we have a specialist in our team

File: docs/content/reference/scanners/_index.md (hunk 2 above)
Comment (VakarisZ) [reply docs/content/reference/scanners/_index.md:8]: Can we just split it up into separate pages and reference what we have?

**Review by cakekoa — APPROVED** [review]

**Review by mssalvatore — APPROVED** [review]

File: docs/content/reference/scanners/_index.md (hunk 2 above)
Comment (ybasford) [reply docs/content/reference/scanners/_index.md:8]: @VakarisZ  Could we take this offline so I can understand what changes we want to make and apply them?

**Pull request merged**